migration/src/androidTest/kotlin/kin/sdk/migration/MigrationManagerIntegrationTest.kt

```diff
@@ -75,13 +85,22 @@ class MigrationManagerIntegrationTest {
 
     private fun removeData() {
         val sharedPreferences = getSharedPreferences()
-        sharedPreferences.edit().remove(KIN_MIGRATION_COMPLETED_KEY).apply()
+        sharedPreferences.edit().clear().apply()
+    }
+
+    @Test
+    @LargeTest
+    fun start_OldKinBlockchain_getOldKinClient() {
+        val oldKinClient = getKinClientOnOldKinBlockchain()
```

Comment:
the test is not really clear (especially without easily being able to see what `getKinClientOnOldKinBlockchain` does)
I don't understand what is under test here, what method we test on the migration module.

Reply:
Maybe the name should be different.
What I test here is that calling start on the old kin blockchain will return an old kin block chain client (and accounts)